### PR TITLE
Feature/jse 45 pallete setup

### PR DESCRIPTION
**Pull request by AubreyMartin**
Feature/JSE-45 Palette Setup

Implemented palette setup in theme.ts with new and updated color definitions. 

Configured primary, secondary, backgroundAlt, secondaryLight, gray, warning, and accent colors.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,117 +1,262 @@
...
+        </Grid>
+        <Grid item xs={3}>
+          <Paper
+            style={{
+              height: 100,
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "bottom",
+              backgroundColor: "#62BCFA", color: "#FFFFFF",
+            }}
```
Comment (judemartin) [inline src/App.tsx:150]: Hey @AubreyMartin , can we pull these colors from the theme.ts

File: src/App.tsx [hunk 2]
```diff
@@ -1,117 +1,262 @@
...
-        diam sit amet mi ullamcorper vehicula{" "}
-      </Typography>
-            </Grid>
-            <Grid item>
-            <Typography variant="h4" className="typography-padding">
-        {" "}
-        H4 - Jost Bold | 14px; 20px Line; -0.2 Spacing <br /> Ut scelerisque
-        hendrerit tellus. Integer sagittis{" "}
-      </Typography>
+        <Grid container spacing={2}>
```
Comment (judemartin) [inline src/App.tsx:169]: instead of div, please use Box

File: src/App.tsx [hunk 3]
```diff
@@ -1,117 +1,262 @@
...
+            
+              Gray <br /> #647196
+           
+          </Paper>
+        </Grid>
+        {/* Row 3 */}
+        <Grid item xs={3}>
+          <Paper
+            style={{
+              height: 100,
```
Comment (judemartin) [inline src/App.tsx:130]: please let's use sx props https://mui.com/system/getting-started/the-sx-prop/

File: src/App.tsx (hunk 2 above)
Comment (AubreyMartin) [reply src/App.tsx:169]: Sure! I'll update the code to use Box instead of div. Thanks for the suggestion!

**Comment by AubreyMartin** [comment]
@judemartin updated the changes 
image

**Review by judemartin — APPROVED** [review]
Great work 💯

File: src/App.tsx [hunk 4]
```diff
@@ -1,117 +1,259 @@
 import { ThemeProvider } from "@mui/material/styles";
 import CssBaseline from "@mui/material/CssBaseline";
 import theme from "./shared-library/theme";
-import Button from "@mui/material/Button";
-import { Stack, Typography,Grid } from "@mui/material";
+import { Button, Stack, Typography, Grid, Paper } from "@mui/material";
 import "./App.css";
+import { Box } from "@mui/system";
```
Comment (judemartin) [inline src/App.tsx:7]: nit : import Box from the same `import { Button, Stack, Typography, Grid, Paper } from "@mui/material"; and not from system

**Pull request merged**